src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/takemanagement/view/TakeManagementView.kt

```diff
@@ -77,54 +71,43 @@ class ViewTakesView : Fragment() {
                     })
             )
         }
-        hbox {
+        vbox {
```

Comment:
I think the hbox is top level, and the vbox is unnecessary?

Reply:
no, the vbox is used to layout 2 things. 
1. the hbox that houses the `next verse`/`previous verse` buttons, along with the selectedTakeCard / drag target. 
2. the `scrollpane` which shows all the alternate takes as well as the `newtake` record card